### DISCUSSION TITLE
Cycle over the links with the Tab key in the Markdown or MarkdownViewer widgets

### DISCUSSION THREAD

**Discussion by VincentDemery** [Textualize/textual#3555] [Ideas]
It would be nice if pressing the `Tab` key with a Markdown or MarkdownViewer widget under focus would cycle over the links, which could be "clicked on" by pressing `Enter`.

**Comment by TomJGooding** [comment]
`Tab` is a bit of a reserved key in Textual as it focuses the next widget. The [inline links](https://textual.textualize.io/styles/links/) aren't widgets so there isn't an easy way to navigate/click them with the keyboard, if that makes sense.

**Reply by VincentDemery** [reply]
Any other key (or combination of keys) would be fine. But I admit that this can be complicated if they cannot get focus.
